migrations/2023-06-06-104440_index_post_url/up.sql

```diff
@@ -0,0 +1 @@
+create unique index idx_post_url on post(url);
```

Comment:
Don't do unique, posts can have the same urls (whether posted to different communities, or reposts, etc.)